src/main/java/com/booster/server/VocabularyEntryService.java

```diff
@@ -16,9 +19,24 @@ public VocabularyEntryDto create(CreateVocabularyEntryInput input) {
 
         vocabularyEntryEntity = vocabularyEntryRepository.save(vocabularyEntryEntity);
 
+        return getVocabularyEntryDto(vocabularyEntryEntity);
```

Comment:
Let's rename the method to `vocabularyEntryDto` and avoid using `get` since it is a Java legacy approach and brings no value to the code?